src/os-freebsd.h

```diff
@@ -1,3 +1,7 @@
+#undef _POSIX_C_SOURCE
+#include <sys/param.h>
+#include <sys/socket.h>
+#include <netinet/in.h>
```

Comment:
In commit message you are saying that you are defining `__BSD_VISIBLE` macro, but it is not defined in this pull request.

Also in commit message you have extra lines author/committer/date/changed files which are always present in commit, see it via command `git show --format=fuller --stat`

Reply:
Fixed, it should be correct now

Reply:
Seems you put some extra duplicate lines `Define __BSD_VISIBLE` and `define __BSD_VISIBLE instead of #undef _POSIX_C_SOURCE` into commit message.

Reply:
fixed